app/src/common/chromium/com/igalia/wolvic/browser/api/impl/BrowserInitializer.java

```diff
@@ -18,15 +17,39 @@
 import org.chromium.content_public.browser.DeviceUtils;
 import org.chromium.ui.base.ResourceBundle;
 
+import java.util.ArrayList;
+
 public class BrowserInitializer {
     static String LOGTAG = SystemUtils.createLogtag(BrowserInitializer.class);
 
     private static final String PRIVATE_DATA_DIRECTORY_SUFFIX = "content_shell";
 
+    private boolean mIsReady = false;
+    private ArrayList<Callback> mCallbacks;
+
+    interface Callback {
+        default void onReady() {}
+    }
+
     public BrowserInitializer(@NonNull Context context) {
+        mCallbacks = new ArrayList<>();
         initBrowserProcess(context);
     }
 
+    public void addCallback(@NonNull Callback callback) {
```

Comment:
I know it's a bit nitpicking but I think it's more coherent with the rest of the code to use `registerCallback` and `unregisterCallback`

Reply:
Make sense. I'll fix it.